client/components/embed-container/index.jsx

```diff
@@ -242,6 +243,43 @@ function embedStory( domNode ) {
 	}
 }
 
+function embedTiledGallery( domNode ) {
+	debug( 'processing tiled gallery for', domNode );
+	const galleryItems = domNode.getElementsByClassName( 'tiled-gallery__item' );
+
+	if ( galleryItems && galleryItems.length ) {
+		const imageItems = Array.from( galleryItems );
+
+		// Replace the gallery with updated markup
+		createRoot( domNode ).render(
+			<div className="gallery-container">
+				{ imageItems.map( ( item ) => {
+					const itemImage = item.querySelector( 'img' );
+					const itemLink = item.querySelector( 'a' );
+
+					const imageElement = (
+						<img
+							id={ itemImage?.id || undefined }
+							className={ itemImage?.className || undefined }
+							alt={ itemImage?.alt || '' }
+							src={ itemImage?.src || undefined }
+							srcSet={ itemImage?.srcSet || undefined }
```

Comment:
Hmm... I am seeing `srcSet` coming through as undefined, but that feels like it's something happening in the DOM rendering, but I think we can keep the option for it being present.